runtime/caches/fileSystem.ts

```diff
@@ -175,12 +122,21 @@ function createFileSystemCache(): CacheStorage {
     }
   }
 
-  const caches: CacheStorage = {
+  const cachesFs: CacheStorage = {
     delete: (_cacheName: string): Promise<boolean> => {
-      throw new Error("Not Implemented");
+      if (
+        fsDir && existsSync(fsDir)
```

Comment:
does it need to be sync?
```suggestion
        fsDir && existsSync(fsDir)
```